### PR TITLE
Karan Kapoor PR

### PR DESCRIPTION
**Pull request by k-kaps**
Hey WARG CV Team, this is Karan Kapoor, I'm an incoming Mechatronics Engineering student. I have been working on the bootcamp for a couple of weeks and got the following results. 
Please let me know your suggestions and advice.
Thanks.

Edit: Also, surprisingly, I was getting a validation loss lower than the training loss initially for the first 2 EPOCHS, any reason why this would happen?

### CODE REVIEW

**Comment by k-kaps** [comment]
[Code Output.txt](https://github.com/UWARG/computer-vision-bootcamp/files/9329219/Code.Output.txt)

**Review by mikashaw — APPROVED** [review]
Hey Karan, congrats on completing the CV bootcamp! I really like how clean and object oriented the code is, I also like how everything runs within a clear pipeline. As for your question regarding lower validation loss than training loss; I believe this tends to occur wen validation and training data have some meaningful differences that just happens to perform slightly better after an epoch of training, another time this happens is when the model is underfit, but looking at your graph I don't believe this to be the case. Great job on your bootcamp and welcome to the team :) Myself of Aydan will ping the CV channel when we figure out when our first meeting is so look out for that
